doc/fileformat/nmodl.rst

```diff
@@ -158,6 +158,45 @@ Arbor-specific features
   made available through the ``v_peer`` variable while the local membrane potential
   is available through ``v``, as usual.
 
+Voltage Processes
+-----------------
+
+Some cases require direct manipulation of the membrane voltage ``v``; which is
+normally prohibited and for good reason so. For these limited application,
+however, we offer mechanisms that are similar to ``density`` mechanism, but are
+tagged with ``VOLTAGE_PROCESS`` where normally ``SUFFIX`` would be used.
+
+This is both a very sharp tool and a somewhat experimental feature. Depending on
+our experience, it might be changed or removed. Using a ``VOLTAGE_PROCESS``,
+voltage clamping and limiting can be implemented, c.f. relevant examples in the
+``default`` catalogue. Example: limiting membrane voltage from above and below
+
+.. code:: none
+
+    NEURON {
+        VOLTAGE_PROCESS v_limit
+        GLOBAL v_low, v_high
+    }
+
+    PARAMETER {
+        v_high =  20 (mV)
+        v_low  = -70 (mV)
+    }
+
+    BREAKPOINT {
+         v = max(min(v, v_high), v_low)
+    }
+
+As of the current implementation, we note the following details and constraints
+
+* only the ``INITIAL`` and ``BREAKPOINT`` procedures are called.
```

Comment:
I'll double check, but I did not currently enable solvers. That doc above is poorly written and `modcc`
should probably be made aware that it's neither allowed to emit `NET_RECEIVE` and `POST_EVENT` blocks,
nor `SOLVE` statements.
I was honestly a bit scared of the power this feature would unleash otherwise, but given the discussion about 
`lif.mod` I should reconsider this.

Reply:
State updates enabled.